docs/howto_firewall.md

```diff
@@ -1,33 +1,40 @@
 # Firewall Configuration
-MicroShift does not require a firewall to run. The following ports should be considered for MicroShift if a firewall is enabled.
+MicroShift does not require a firewall for normal operation. If a firewall is enabled, it should be configured according to the requirements in this document.
```

Comment:
```suggestion
MicroShift does not require a firewall for normal operation, although it's recommended to avoid unwanted access to the MicroShift API. If a firewall is enabled, it should be configured according to the requirements in this document.
```

Reply:
Done